sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -83,12 +83,10 @@ object ScalaNativePluginInternal {
   )
 
   lazy val scalaNativeConfigSettings: Seq[Setting[_]] = Seq(
-    nativeTarget := interceptBuildException {
-      val cwd   = nativeWorkdir.value.toPath
-      val clang = nativeClang.value.toPath
-      Discover.targetTriple(clang, cwd)
+    nativeTarget := { // placeholder for future use
+      ""
```

Comment:
by default it is empty string, isn't it?

Also I feel that `Option[String]` for this case is more semantic.

Reply:
Totally agree but that should probably be addressed in a PR that allows cross compiling.

Reply:
I feel that you may remove it here. Can't you?

Reply:
I think people may want to use `-target` vs `-arch` but we don't want rip out `-target` if we may need it. If this PR does the job for you, it is much smaller and we are not preventing the other PRs either.

Reply:
Let close this discussion because it is style question.

Reply:
You mean `-arch` versus `-target`?

Reply:
@ekrich I mean to remove `nativeTarget` with empty string because it is empty string by default.

But I might be wrong and it is the place where this key has it's default value.

Reply:
Oh ok, `sbt` will not allow the internal `nativeTarget` to be unassigned. I tried to remove that assignment/block and the build will not load. There might be some other way but this seemed the best for now and it leaves a placeholder and a place to experiment for now.

Reply:
@ekrich it is clear. Thanks.